release_docs/README_HDF5_CMake

```diff
@@ -6,17 +6,18 @@ This tar file contains
     CTestScript.cmake
     HDF5config.cmake    CMake scripts for building HDF5
     HDF5options.cmake
-    hdf5-1.10.7         HDF5 1.10.7 source
+    hdf5-1.10.8         HDF5 1.10.8 source
```

Comment:
Can the version numbers in this file be replaced by some generic phrase referring to the current version?  If not, h5vers should update them, or they should  somehow be inserted by the build process?

Reply:
1.10.x